Libs/MRML/Core/vtkMRMLVolumeArchetypeStorageNode.h

```diff
@@ -82,6 +83,13 @@ class VTK_MRML_EXPORT vtkMRMLVolumeArchetypeStorageNode : public vtkMRMLStorageN
   /// instance to turn off compression.
   virtual void ConfigureForDataExchange();
 
+  ///
+  /// Provide a uniform way to populate the volume nodes's itk
+  /// metadatadictionary from the reader.  Since itk::MetaDataDictionary
+  /// is not exposed in python, this method allows it to be set indirectly
+  /// using only wrapped types.
+  static void SetMetaDataDictionaryFromReader(vtkMRMLVolumeNode*, vtkITKArchetypeImageSeriesReader*);
```

Comment:
Just a side note - I didn't know about this feature, or where it is used, but I searched, and it seems that it is used in MRMLIDIO, and specifically for communicating diffusion-related attributes: https://github.com/Slicer/Slicer/blob/master/Libs/MRML/IDImageIO/itkMRMLIDImageIO.cxx#L313-L328. I wonder if this is working, or if anyone is using it. @ihnorton FYI!